src/pages/evm/wallet/SendPage.vue

```diff
@@ -1,21 +1,577 @@
+        viewTokenContract() {
+            if (this.token) {
+                const explorerUrl = ant.stores.chain.loggedEvmChain?.settings.getExplorerUrl();
+                if (explorerUrl) {
+                    window.open(explorerUrl + '/address/' + this.token.address, '_blank');
+                    return;
+                } else {
+                    ant.config.notifyErrorHandler(this.$t('settings.no_explorer', { network: ant.stores.chain.currentChain?.settings.getNetwork() }));
+                }
+            }
+        },
+        goBack() {
+            this.$router.back();
+        },
+        startTransfer() {
+            const token = this.token;
+            const amount = this.finalTokenAmount;
+            const to = this.address;
+            if (this.isFormValid) {
+                ant.stores.balances.transferTokens(token as Token, to, amount).then((trx: TransactionResponse) => {
+                    const chain_settings = ant.stores.chain.loggedEvmChain?.settings;
+                    if(chain_settings) {
+                        ant.config.notifySuccessfulTrxHandler(
+                            `${chain_settings.getExplorerUrl()}/tx/${trx.hash}`,
+                        );
+                    }
+                }).catch((err) => {
+                    if (typeof err.message === 'string') {
+                        console.error(err.message, err.payload, err);
+                        ant.config.notifyFailedTrxHandler(this.$t(err.message));
+                    } else if (typeof err === 'string') {
+                        ant.config.notifyFailedTrxHandler(this.$t(err));
+                    } else {
+                        ant.config.notifyErrorHandler(this.$t('evm_wallet.general_error'));
+                    }
+                });
+            } else {
+                ant.config.notifyErrorHandler(this.$t('evm_wallet.invalid_form'));
+            }
+        },
+    },
+
 });
 </script>
 
 <template>
 <AppPage>
-    <p class="text-black">Send page</p>
+    <template v-slot:header>
+        <div class="c-send-page__title-container">
+            <p class="c-send-page__title"> {{ $t('evm_wallet.send') }}</p>
+        </div>
+        <q-btn
+            class="c-send-page__back-button"
+            flat
+            dense
+            label="Back"
+            icon="arrow_back_ios"
+            @click="goBack"
+        />
+    </template>
+
+    <div class="c-send-page__form-container">
+        <q-form
+            class="c-send-page__form"
+        >
+            <div class="c-send-page__row c-send-page__row--1 row">
+                <div class="col">
+                    <q-input
+                        v-model="address"
+                        outlined
+                        autogrow
+                        :type="useTextarea ? 'textarea' : 'text'"
+                        :label="$t('evm_wallet.receiving_account')"
+                        :rules="[val => !!val || $t('evm_wallet.account_required')]"
```

Comment:
rules should include address validation so the user gets feedback when they have entered an invalid address. also lets add a placeholder

```vue
<q-input
    v-model="address"
    outlined
    autogrow
    :placeholder="'0x'.concat('0'.repeat(40))"
    :type="useTextarea ? 'textarea' : 'text'"
    :label="$t('evm_wallet.receiving_account')"
    :rules="[
        val => !!val || $t('evm_wallet.account_required'),
        val => /^0x[a-fA-F0-9]{40}$/.test(val) || 'Invalid address'
    ]"
/>
```

(but add the invalid address to i18n)

Reply:
made an issue for this, you can ignore for this PR if you want
https://github.com/telosnetwork/telos-wallet/issues/263